src/command_modules/azure-cli-appservice/HISTORY.rst

```diff
@@ -4,6 +4,10 @@ Release History
 ===============
 (unreleased)
 +++++++++++++++++++
+* webapp: Add generate container CI URL for Web App
+
+(unreleased)
```

Comment:
'unreleased' appears twice.
Please combine the new addition to the already existing header.